src/app/modal/components/lose-changes/lose-changes.component.scss

```diff
@@ -0,0 +1,29 @@
+@import "node_modules/govuk-frontend/govuk/all";
+
+.mat-dialog-content {
+  overflow: hidden;
+}
+
+.title {
+  color: #0B0C0C;
```

Comment:
Can we use a color from govuk instead please?
$govuk-text-colour should work

(for reference; https://design-system.service.gov.uk/styles/colour/)

Reply:
removed the title class. It slightly differs the design but this is something that we can agree with testers and BAs.